app/models/like.rb

```diff
@@ -1,4 +1,10 @@
 class Like < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  after_save :updates_post_likes_counter
+
+  def updates_post_likes_counter
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
[OPTIONAL]
- [ ] Same for the method `updates_post_likes_counter` it would be nice to make it private, since it will be call every time that a new like has been save in the database